src/main/java/org/killbill/billing/plugin/adyen/api/mapping/UserDataMappingService.java

```diff
@@ -69,8 +73,79 @@ public static UserData toUserData(@Nullable final Account account, final Iterabl
         // set ip
         userData.setShopperIP(PluginProperties.findPluginPropertyValue(AdyenPaymentPluginApi.PROPERTY_IP, properties));
 
-        return userData;
-    }
+        //determine AccountAge Indicator
+        final String accountAgeIndicator = PluginProperties.findPluginPropertyValue(ACCOUNT_AGE_INDICATOR,properties);
+        final Optional<String> optionalAccountAgeIndicator = toLastName(accountAgeIndicator,account);
+        final String accAgeIndicator = optionalAccountAgeIndicator.isPresent() ? optionalAccountAgeIndicator.get() : null;
+        userData.setAccountAgeIndicator(accAgeIndicator);
+
+        final String accountChangeDate = PluginProperties.findPluginPropertyValue(ACCOUNT_CHANGE_DATE,properties);
+        if(accountChangeDate != null) {
+            userData.setAccountChangeDate(DateTime.parse(accountChangeDate));
+        }
+        final String accountChangeIndicator = PluginProperties.findPluginPropertyValue(ACCOUNT_CHANGE_INDICATOR,properties);
+        userData.setAccountChangeIndicator(accountChangeIndicator);
```

Comment:
This value can be backfilled by using `account.getCreatedDate()` and `account.getUpdatedDate()`. Also, since Adyen expects a specific set of values (`thisTransaction`, `lessThan30Days`, `from30To60Days`, `moreThan60Days`), an Enum would be appropriate here.